src/client.c

```diff
@@ -222,9 +223,9 @@ int word_difference(char *a, char *b) {
   if (a == b)
     return 0;
   int diff = 0;
-  for (size_t i = 0; i < min(strlen(a), strlen(b));
-       i++, diff += abs(a[i] - b[i]))
-    ;
+  for (size_t i = 0; i < min(strlen(a), strlen(b)); i++) {
```

Comment:
this loop might be sped up a lot by caching the 2 strlen in size_t variables, so they dont need to be called on every iteration

Reply:
@rofl0r I don't think performance is important in that place.
@BachoSeven You will need to fix your PKGBUILDs for this project after I merged the new commits

Reply:
@braph Thanks, I will

Reply:
although, I just tried to build my PKGBUILD for `nbfc-linux-git` and it seems fine, what would need to be tweaked?

Reply:
@BachoSeven The PKGBUILD has PREFIX as /usr instead of /usr/local and is unable to start. But when changed to /usr/local, it gives an error while installing as `nbfc-linux-git: /usr/local/share/man exists in filesystem (owned by filesystem)`. Can you check it out?

Reply:
@msourabh24 PREFIX should definitely be `/usr` in the PKGBUILD, my mistake in the PKGBUILD was the omission of `confdir` as suggested by @rofl0r in https://github.com/nbfc-linux/nbfc-linux/pull/27#issuecomment-975768750.

It should be fixed now.

Reply:
Actually, using `make PREFIX=/usr confdir=/etc DESTDIR="${pkgdir}" install` __almost__ does the right thing, meaning that in the resulting package `/etc` is created instead of `/usr/local/etc`; however the `nbfc_service` binary still has `/usr/local/etc`... how can I change that @rofl0r?

Reply:
Nevermind, I had omitted the parameters in `build()`.

Reply:
Yeah, it works now. Thanks for the quick fix!